### ISSUE TITLE
Cannot connect to caddx vista in configurator

### ISSUE THREAD

**Issue by leoponce82** [fpv-wtf/wtfos-configurator#239]
My caddx vista does not show up on the configurator (HOME tab, connect to device). When I go to ROOT tab, I can see a "CDC Serial interface" but it does not connect. (It does not always show up on the root tab)

I tried using develop.fpv.wtf, and I can see the vista on the devices list "DJI Fpv Air unit Lite" but it gives an error:
"Could not connect to the device, make sure that ADB server is not running on your machine (adb kill-server) and that you are not connected in another tab/window"

I tried the command, still nothing. 

Please help and thanks in advance

v1.0.0 - Trace ID: 5af43bbb-6b74-489c-9185-ac06c25be1ea

**Comment by stylesuxx** [comment]
@leoponce82 since you closed it, what was the issue?

**Comment by leoponce82** [comment]
My caddx vista was not recognized by DJI assistant 2 (FPV series). I appeared in "devices and printers" on windows, and sometimes the folder (where naco.txt was) would appear, but not consistently.  (On Windows device manager it would appear as Bulk interface)
I tried rooting it but the fpv.wtf configurator would not recognize it. On develop.fpv.wtf the vista was recognized but it would not connect. 
Ultimately, I changed the driver using Zadig. I went fpv.wtf and rooted it successfully.

**Comment by robertrypula** [comment]
@leoponce82 I think I have the same issue as you had. I have both googles v2 and vista rooted and everything was working fine. I was playing with OSD placement in Betaflight configurator and after few restarts I started to see "Waiting for OSD..." in the googles

Currently I don't see Vista in WTF configurator. Vista mass storage is showing up. DJI Assistant doesn't see Vista at all. In Zadig I see this:

![image](https://user-images.githubusercontent.com/6407731/205455227-11559ac7-7133-48c6-8fad-71aadf6b8f6a.png)

On the left is what I see when Vista is connected (I marked potential Vista devices with orange dots), on the right when it's not. What device you selected and what option you picked to solve the issue?

![image](https://user-images.githubusercontent.com/6407731/205455329-0aaca32a-f0d3-4dc8-b6b2-f9634508e777.png)

**Comment by leoponce82** [comment]
@robertrypula my issue was a little different since I los the "waiting for OSd" but the "not seeing vista in the configurator and assistant " behaviour is the same. 
I used the Bulk interface and replaced the driver with zadig. I did it two times. First to "USBlib" and then i think it was "winUSb".

**Comment by robertrypula** [comment]
I figured that out! I was changing many different combinations of the devices and drivers and looks like the device that WTF configurator is using is just "CDC Serial" (Interface 4 on my laptop). The driver that we need to install is USB Serial (CDC). From the dropdown I picked first option "Install Driver":

![image](https://user-images.githubusercontent.com/6407731/205457100-9e5c0bc0-9c21-4e25-9051-4ce74225666c.png)

I needed to root again. I had also problem with Health Checks:

![image](https://user-images.githubusercontent.com/6407731/205457234-53f2bc22-9d7d-44e4-8896-ad621cb5c05a.png)

![image](https://user-images.githubusercontent.com/6407731/205457235-0beed58f-400a-4fe3-8933-dd9848ec56d2.png)

At the end everything works as expected. There is only one bad feeling in my head that the issue will happen again. I have CineLog35 and I'm using 4S lipos. Vista is connected directly to the VBat and I almost always have arcing noise then I connect battery. In the other threads I read that it could be the cause of loosing root and (in my case) ending in "invisible" vista.
